shims/spark320/src/main/scala/com/nvidia/spark/rapids/shims/spark320/Spark320Shims.scala

```diff
@@ -510,8 +538,14 @@ class Spark320Shims extends Spark32XShims {
     GpuOverrides.scan[OrcScan](
       "ORC parsing",
       (a, conf, p, r) => new ScanMeta[OrcScan](a, conf, p, r) {
-        override def tagSelfForGpu(): Unit =
+        override def tagSelfForGpu(): Unit = {
           GpuOrcScanBase.tagSupport(this)
+          // we are being overly cautious and that Orc does not support this yet
+          if (a.isInstanceOf[SupportsRuntimeFiltering]) {
```

Comment:
here and elsewhere: Does calling tagSupport(this) on L542 make sense if  this is true?

Reply:
Per my understanding. Plugin is trying best to explain **all the reasons** that won't run on GPU at a time.